.obs/workflows.yml

```diff
@@ -16,16 +16,12 @@ pr:
         - target_project: devel:openQA
           target_repository: openSUSE_Tumbleweed
         architectures: [ x86_64 ]
-      - name: openSUSE_Leap_15.4
+      - name: openSUSE_Leap_15.5
```

Comment:
> To use the most recently released Leap 15.5 as CI container base in os-autoinst and openQA we should adapt that and also run the according OBS checks on Leap 15.4

I would think we want both? Quoting from the ticket.

Reply:
From the very same ticket 😉 
> If it's easier just replace the version

Reply:
Two votes for just going with 15.5 on the daily so I guess why not